internal/venice-common/src/main/java/com/linkedin/venice/kafka/protocol/enums/MessageType.java

```diff
@@ -21,7 +21,8 @@
  */
 public enum MessageType implements VeniceEnumValue {
   PUT(0, Constants.PUT_KEY_HEADER_BYTE), DELETE(1, Constants.PUT_KEY_HEADER_BYTE),
-  CONTROL_MESSAGE(2, Constants.CONTROL_MESSAGE_KEY_HEADER_BYTE), UPDATE(3, Constants.UPDATE_KEY_HEADER_BYTE);
+  CONTROL_MESSAGE(2, Constants.CONTROL_MESSAGE_KEY_HEADER_BYTE), UPDATE(3, Constants.UPDATE_KEY_HEADER_BYTE),
+  CONTROL_MESSAGE_DIV(4, Constants.DIV_KEY_HEADER_BYTE);
```

Comment:
This is a good question. The idea in current implementation is to reuse the existing chunking mechanism as much as we could. To reuse the `CONTROL_MESSAGE` category, we got to be backward compatible with the current protocol which probably indicates the more code modifications:

1. KME avro definition evolution. 
2. Chunking is only supported for PUT message as of today and all related codes and I would image that to make it work for ControlMessage type, it would require more works. (see the doc "Enable Chunking for Control Messages" for a bit more context)

Introduce a new type `CONTROL_MESSAGE_DIV` allows us to choose the KME format (especially the payloadUnion) freely, thus giving us the flexibility to reuse almost all the existing mechanism for chunking support.